app/Jobs/Warnings/WarningFetcher.php

```diff
@@ -10,10 +10,10 @@
 use Illuminate\Foundation\Bus\Dispatchable;
 use Illuminate\Queue\SerializesModels;
 use Illuminate\Support\Arr;
-use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Cache;
```

Comment:
Don't use the facade. Just inject a `Cache` instance in the constructor. See https://github.com/vostpt/api/blob/master/app/Http/Controllers/Controller.php#L33